Drop analytics_id option if VERILATOR_AUTHOR_SITE environment variable isn't set

aelmahmoudy opened the pull request:
The inclusion of gtags.js causes Debian lint tool to report privacy-breach-generic tag

wsnyder commented:
Please merge master in, so you won't need your CONTRIBUTOR fix.

That was added for the Verilator web site build itself, but I agree shouldn't be in other places.  Can you add a environment variable to set it (that defaults to not being set) so I can still use it on the main site (by setting the env var)?

aelmahmoudy commented:
Could you give me some example code to follow ?

aelmahmoudy commented:
Nevermind I found an example in conf.py itself. But didn't test it.

aelmahmoudy commented:
Updated the patch to use the VERILATOR_AUTHOR_SITE environment variable, the same variable used to enable long tests.
I also added '--enable-ganalytics' to configure script.
I also tested the build with the env. var. being set and not set.

wsnyder commented:
Thanks, I committed it slightly differently based on your ideas, as I think most authors really don't want the analytics.
